packages/source-filesystem/src/index.ts

```diff
@@ -1,7 +1,15 @@
-import { defaultsDeep } from 'lodash-es';
-import { read } from 'to-vfile';
-
-import type { LoadedFlatbreadConfig, SourcePlugin } from '@flatbread/core';
+import slugify from '@sindresorhus/slugify';
+import { defaultsDeep, merge } from 'lodash-es';
+import { read, write } from 'to-vfile';
+import ownPackage from '../package.json' assert { type: 'json' };
```

Comment:
woah I've never seen this before, TIL about this new funky Node change

Reply:
Yeah I didn't know about it either until AVA started yelling about it